src/lab1/Main.java

```diff
@@ -0,0 +1,176 @@
+package lab1;
+import java.util.Scanner;
+
+public class Main {
+    static void showMenu() {
+        System.out.println("Press 1 to input movies\n" +
+                "Press 2 to display movies and ratings\n" +
+                "Press 3 to input more movies\n" +
+                "Press 0 to exit\n" +
+                "Press 4 to find statistics\n" +
+                "Press 5 to search for a movie\n" +
+                "Press 6 to update movie ratings\n" +
+                "Press 7 to delete a movie\n" +
+                "Press 8 to sort movies\n" +
+                "Enter your choice: ");
+    }
+
+    static Movie[] addMovies(int n, Scanner sc, Movie[] movies) {
+        Movie[] newMovies = new Movie[movies.length + n];
+        System.arraycopy(movies, 0, newMovies, 0, movies.length);
+        for (int i = movies.length; i < newMovies.length; i++) {
+            sc.nextLine();
+            System.out.println("Enter the name of movie " + (i + 1) + ": ");
+            String movieName = sc.nextLine();
+            System.out.println("Enter the rating of movie: ");
+            double movieRating = sc.nextDouble();
+            newMovies[i] = new Movie(movieName, movieRating);
+        }
+        return newMovies;
+    }
+
+    static double calcAvgRating(Movie[] movies) {
+        double avgRating = 0;
+        for (Movie movie : movies) {
+            avgRating += movie.getRating();
+        }
+        avgRating = avgRating / movies.length;
+        return avgRating;
+    }
+
+    static void findMaxAndMinRating(Movie[] movies) {
+        double maxRating = movies[0].getRating();
+        double minRating = movies[0].getRating();
+        for (Movie movie : movies) {
+            if (movie.getRating() > maxRating) {
+                maxRating = movie.getRating();
+            }
+            if (movie.getRating() < minRating) {
+                minRating = movie.getRating();
+            }
+        }
+        System.out.println("The movie with the maximum rating: " + maxRating);
+        System.out.println("The movie with the minimum rating: " + minRating);
+    }
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        Movie[] movies = new Movie[0];
```

Comment:
instead of sending movies and sc to each method you can declare these in class level